manifests/addons/metrics-emitter.yml

```diff
@@ -0,0 +1,40 @@
+instance_groups:
+- name:      rmq-broker
+  jobs:
+  - name: loggregator_agent
+    release: loggregator-agent
+    consumes:
+      doppler:
+        from: doppler
+        deployment: (( grab meta.cf.deployment_name ))
+    properties:
+      loggregator:
+        tls:
+          ca_cert: (( vault meta.cf.exodus_path ":loggregator_ca" ))
+          agent:
+            cert: (( vault meta.cf.exodus_path ":loggregator_tls_agent_cert" ))
+            key: (( vault meta.cf.exodus_path ":loggregator_tls_agent_key" ))
+  - name: rabbitmq-metrics-emitter
+    release: rabbitmq-metrics-emitter
+    properties:
+      rabbitmq_metrics_emitter:
+        cloud_foundry:
+          api:    (( grab meta.cf.api_url ))
+          skip_ssl_validation: (( grab meta.cf.skip_ssl_validation ))
+          username:   (( vault meta.cf.exodus_path ":admin_username" ))
+          password:   (( vault meta.cf.exodus_path ":admin_password" ))
+        rmq_management:
+          endpoint: (( concat "http://" params.rmq_domain ":15672/api" ))
```

Comment:
the management API listens on https 15671, with a kit feature to listen on http 15672 now.

You can use `meta.mgmt_port` to get the port that mgmt is listening on, and `mgmt.mgmt_scheme` to get the url scheme to specify. You'll need to re-rebase off master to get the latter property.

Reply:
To further support talking to the management API, a ca cert option for communicating to the management API would be useful.